### PR TITLE
fix(community-pagination): align with current design

### PR DESCRIPTION
**Pull request by marcod1419**
This PR is targeted towards the original Pagination fix PR to fully align with design and simplify the render logic slightly. Once this is approved, we may want to squash Enrico's pagination in progress commit (a15be72) into mine to make things cleaner.

### CODE REVIEW

File: packages/Pagination/Pagination.jsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React, { Component } from 'react'
```
Comment (agorovyi) [inline packages/Pagination/Pagination.jsx]: I'm wondering if this component can also be refactored as a functional component with hooks as part of this PR?

File: packages/Pagination/Pagination.md [hunk 2]
```diff
@@ -4,7 +4,19 @@
...
+- You must pass in a value to `copy` in order to render helpful text to assistive technology users
+
+```jsx
+<Pagination copy="en">
+  <Pagination.Panel>Content 1: any type of content can be used here.</Pagination.Panel>
+  <Pagination.Panel>Content 2: use responsibly.</Pagination.Panel>
+  <Pagination.Panel>Content 3: Pagination Panels take &lt;div /&gt; tags, image tags, heading tags and so on.</Pagination.Panel>
+</Pagination>
+<Box inset={4}> </Box>
+```
```
Comment (agorovyi) [inline packages/Pagination/Pagination.md]: Do we need to show two similar code examples? Existing example also demonstrates a `copy` prop.

File: packages/Pagination/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (agorovyi) [inline packages/Pagination/package-lock.json]: This file shouldn't be committed.

File: packages/SideNavigation/package-lock.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (agorovyi) [inline packages/SideNavigation/package-lock.json]: This file shouldn't be committed.

File: packages/SkeletonProvider/package-lock.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (agorovyi) [inline packages/SkeletonProvider/package-lock.json]: This file shouldn't be committed.

File: packages/Pagination/Pagination.md (hunk 2 above)
Comment (marcod1419) [reply packages/Pagination/Pagination.md]: Agreed, this can probably be removed.

File: packages/Pagination/Pagination.jsx (hunk 1 above)
Comment (marcod1419) [reply packages/Pagination/Pagination.jsx]: I think this is out of scope for now. The main purpose of this PR is to move the functionality closer to the desired design. Refactoring for hooks could increase potential complexity and make the process longer.

File: packages/Pagination/package-lock.json (hunk 3 above)
Comment (marcod1419) [reply packages/Pagination/package-lock.json]: Currently in community, the lock files will continually appear due to unmatching dependencies. For now I think we're better off just committing these.

File: packages/SideNavigation/package-lock.json (hunk 4 above)
Comment (marcod1419) [reply packages/SideNavigation/package-lock.json]: Currently in community, the lock files will continually appear due to unmatching dependencies. For now I think we're better off just committing these.

File: packages/SkeletonProvider/package-lock.json (hunk 5 above)
Comment (marcod1419) [reply packages/SkeletonProvider/package-lock.json]: Currently in community, the lock files will continually appear due to unmatching dependencies. For now I think we're better off just committing these.

**Pull request merged**